src/types.mligo

```diff
@@ -117,10 +117,8 @@ type voter =
   }
 
 // Amount of `seconds` used for lengths of time
-type seconds = nat
-
-// Length of a 'stage'
-type period = { length : seconds }
+type levels = { levels : nat }
```

Comment:
Is `level` a reserved word?

`levels` sounds incorrect in some uses of the type, maybe we should use `blocks`?

Reply:
Changed